content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Collect Google Cloud Logs with a Pub/Sub Push Subscription
+kind: documentation
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "/logs/explorer/#visualize"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "/logs/log_configuration/processors"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+---
+
+<div class="alert alert-danger">
+This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Use a <strong>Pull</strong> subscription with the Datadog Dataflow template to forward your Google Cloud logs to Datadog instead. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
+</div>
+
+## Overview
+
+This guide describes how to forward logs from your Google Cloud services to Datadog through a **Push** subscription to a [Google Cloud Pub/Sub][10] topic.
+
+To collect logs from applications running in GCE or GKE, you can also use the [Datadog Agent][13].
+
+**Note**: If you have a [Google Cloud VPC][14] in your Google Cloud environment, the **Push** subscription cannot access endpoints outside the VPC.
+
+## Setup
+
+### Prerequisites
+
+The [Google Cloud Platform integration][1] is successfully installed.
+
+### Create a Cloud Pub/Sub topic
+
+1. Go to the [Cloud Pub/Sub console][2] and create a new topic.
+
+    {{< img src="integrations/google_cloud_platform/create_a_topic.png" alt="Create a topic" style="width:80%;">}}
+
+2. Give that topic an explicit name such as `export-logs-to-datadog` and click _Create_.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][3]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [the Monitor the Log Forwarding section](#monitor-the-log-forwarding) for information on setting up monitor notifications if you approach those limits.
+
+### Subscribe to your Google Cloud Pub/Sub topic
+
+1. In the [Cloud Pub/Sub console][2], select `Subscriptions` in the left hand navigation. Click `Create Subscription`.
+2. Create a subscription ID and select the topic you previously created.
+3. Select the `Push` method and enter the following command, replacing `<DATADOG_API_KEY>` with the value of a valid [Datadog API key][4]: 
+```
+https://gcp-intake.logs.{{< region-param key="dd_site" code="true" >}}/api/v2/logs?dd-api-key=<DATADOG_API_KEY>&dd-protocol=gcp
+```
```

Comment:
Not sure if it's worth mentioning, but it always throws me off that changing the Site dropdown changes this URL. 🤷

Reply:
Thanks for bringing that up, is there anything in particular that throws you off about that? 
It's not in my power to change that behavior, but I can follow up with the websites team or make adjustments here if you think it might be improvable.

Reply:
Understood! My concern is that customers don't know they should change the site dropdown before copying the URL to use. At least it wasn't obvious to me.

Don't have any specific solution besides maybe a note about the behavior somewhere, but at least this is consistent everywhere. 👍

Reply:
Thanks for clarifying, added a note in [59366e9](https://github.com/DataDog/documentation/pull/19467/commits/59366e9e72ea802976785604447e1f3b44792225)